RELEASE.md

```diff
@@ -0,0 +1,88 @@
+# Release 0.2.0
+
+# Performance Optimization
+
+## GPU stitch fusion
```

Comment:
Should we mention cpu stitch as well? I am not sure if we mentioned this feature in previous release notes.

Reply:
CPU stitch is already released when we open-source BladeDISC project. If we want to announce features developed before open-source the release note of v0.2.0, we may need a rule to select the features from all the existing techniques.

Reply:
@wyzero I have attached the link about the materials of CPU stitch optimizations.